### ISSUE TITLE
First instance is unavailable after lock or sleep

### ISSUE THREAD

**Issue by joshcangit** [lanoxx/tilda#401]
#### OS:	Ubuntu 18.04.3 LTS

This should probably be in Issue #294 but a new issue should bring more attention to this very problematic **bug**.

Whenever my computer is locked, sleeping or hibernating, the first instance of Tilda will be stuck, completely unable to be reopened, running or idling in the background as shown in System Monitor. Even after logging back in, there's no way to enter that first instance other than killing that process and then opening Tilda again.

![gnome-system-monitor](https://user-images.githubusercontent.com/33022160/69438346-c0ad4080-0d3c-11ea-9ceb-ed7b8558084a.png)

One solution would be implementing Issue #145 and also allowing **only 1 instance** of Tilda. This bug makes running `apt-get update` very risky.

**Comment by lanoxx** [comment]
Could you try to reproduce this with the latest version of tilda in the master branch?

**Comment by joshcangit** [comment]
Yes, this **bug** is still there. I'll try to show more details when I'm on my computer.

**Comment by joshcangit** [comment]
I'm having trouble screen recording the process for Tilda. The screen recording ends when locked or sleep.
